gui/src/components/dashboard/capacity-gauge.vue

```diff
@@ -61,21 +61,24 @@ import * as c3 from "c3";
 export default class CortxCapacityGauge extends Vue {
 
   public usedLegendClass = "capacity-used-green";
-  public chartDataVal: number;
+  public chartDataVal;
   public created() {
     const demoData = [["x", 0]];
     const capacityRes = this.$store
       .dispatch("performanceStats/getCapacityStats")
       .then(capacityC3Data => {
-        this.chartDataVal = capacityC3Data[0][1] ? capacityC3Data[0][1] : 0.00;
-        if (this.chartDataVal <= 50.00) {
-          this.usedLegendClass = "capacity-used-green";
-        }
-        if (this.chartDataVal > 50.00) {
-          this.usedLegendClass = "capacity-used-orange";
-        }
-        if (this.chartDataVal >= 90.00) {
-          this.usedLegendClass = "capacity-used-red";
+        if (capacityC3Data) {
+          this.chartDataVal = capacityC3Data[0][1] ? capacityC3Data[0][1] : 0.00;
```

Comment:
why do we want to check for 0.00 ot 50.00 or so on.. we should be using simple integer. for example 0,50 etc. conversion will happen automatically

Reply:
resolved the above review comment, checked only for numbers